packages/espressocash_app/lib/features/activities/data/transaction_repository.dart

```diff
@@ -46,24 +96,27 @@ class TransactionRepository {
     final query = _db.select(_db.transactionRows)
       ..where((tbl) => tbl.id.equals(id));
 
-    return query.watchSingle().asyncExpand((row) => _match(row.toModel()));
+    return query.watchSingle().asyncMap(
+          (row) async => row.toModel().then((value) => _match(value).first),
+        );
   }
 
   Future<void> saveAll(
     Iterable<TxCommon> txs, {
-    required bool clear,
+    bool clear = false,
```

Comment:
I do not understand purpose of this parameters, if they are not being set anywhere and always default